### ISSUE TITLE
SARIF File - Default location uri is invalid 

### ISSUE THREAD

**Issue by rseeton** [jfrog/jfrog-cli#2303]
### Describe the bug

I see that the artifactLocation has been added to the 2.51.1 package, producing the following in each results element:

```
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package Descriptor"
                }
              }
            }
          ]
        }
```

The file does pass the SARIF Validation, including the GitHub Ingestion rules, which is good.
However, none of these entries are populated, resulting in the default "Package Descriptor" entry. The default value triggers an 'invalid URI' message, preventing the SARIF file from being loaded by GHAS:

```
Error: Code Scanning could not process the submitted SARIF file:
an invalid URI was provided as a SARIF location: parse "file://Package Descriptor": invalid character " " in host name
```
(It looks like the 'space' character in Package Descriptor is the issue - replacing this with Package_Descriptor resolves the URI validation issue)

### Current behavior

SARIF Files generated have invalid location URI
```
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "file://Package Descriptor"
                }
              }
            }
          ]
        }
```

 The default value triggers an 'invalid URI' message, preventing the SARIF file from being loaded by GHAS:

```
Error: Code Scanning could not process the submitted SARIF file:
an invalid URI was provided as a SARIF location: parse "file://Package Descriptor": invalid character " " in host name
```

### Reproduction steps

Generate SARIF File
Attempt to process SARIF file with ingestor (we use GitHub Advanced Security)

### Expected behavior

Default Location file should be a 'valid' URI that can be parsed.

(It looks like the 'space' character in Package Descriptor is the issue - replacing this with Package_Descriptor resolves the URI validation issue)

### JFrog CLI version

jf version 2.51.1

### Operating system type and version

Centos8

### JFrog Artifactory version

version: 7.55.10, revision: 75510900

### JFrog Xray version

xray_version:3.65.2 xray_revision: bca527a

**Comment by attiasas** [comment]
Closing this as it is a duplicate of the following issue: https://github.com/jfrog/jfrog-cli/issues/2270.

**Comment by rseeton** [comment]
hmmm... we appear to still be waiting on a build with the fixed URI using the new jfrog-cli-core package ( https://github.com/jfrog/jfrog-cli-core/releases/tag/v2.46.2 )

Issue: https://github.com/jfrog/jfrog-cli/issues/2270 is more general issue/request
